packages/moonstone/Dropdown/Dropdown.js

```diff
@@ -133,7 +133,11 @@ const DropdownList = Skinnable(
 					select="radio"
 					selected={selected}
 				>
-					{children}
+					{children.map((o, i) => {
```

Comment:
This may be better to place in the `computed` block.

Reply:
Agree. Additionally, the computed block would make it a bit easier to manage `null` which will throw an error now.

Reply:
Also, let's add a `filter` before the map to guard against `null` array members.

Reply:
Yeap! moved to `computed` block and also added a `filter` to guard `null`